src/Helper/ValidateStreet.php

```diff
@@ -20,7 +20,7 @@ class ValidateStreet
         '(?P<number_suffix>' .
         '[a-z]{1}-?\d{1,3}|' .            // Numbers suffix starts with a letter with optional - followed by numbers or
         '-\d{1,4}|' .                     // starts with - and has up to 4 numbers or
-        '\d{2}\w{1,2}|' .                 // starts with 2 numbers followed by letters or
+        '\d{2}\w{1,2}?|' .                // starts with 2 numbers followed by optional letters or
```

Comment:
Ik denk dat dit niet doet wat je denkt

- `\w` is niet een letter, maar `[A-z\d_]`
- `?` maakt de `{1,2}` "lazy", dat betekent dat hij maar 1 karakter zal matchen, zelfs als er 2 zijn.